src/ports/postgres/modules/deep_learning/test/madlib_keras_fit.sql_in

```diff
@@ -40,7 +40,7 @@ SELECT madlib_keras_fit(
     'model_arch',
     1,
     $$ optimizer=SGD(lr=0.01, decay=1e-6, nesterov=True), loss='categorical_crossentropy', metrics=['mae']$$::text,
-    $$ batch_size=2, epochs=1, verbose=0 $$::text,
+    $$ batch_size=2, epochs=1, verbose=0, callbacks=[TensorBoard(log_dir='/tmp/tensorflow/single/')] $$::text,
```

Comment:
can we assert that the directory `/tmp/tensorflow/single/` is not empty or contains expected files ?

Reply:
That is the job of TensorBoard, I don't think we should get involved.

Reply:
yes agree this is a tensorboard thing, and user needs to manage the directories themselves where the log files are written